test/utils/deploy-minikube.sh

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+export CHANGE_MINIKUBE_NONE_USER=true
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+KUBECONFIG=$HOME/.kube/config
+KUBERNETES_VERSION=v1.18.1
+
+# install and start minikube cluster
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.18.1/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.1/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+mkdir -p $HOME/.kube $HOME/.minikube
+touch $KUBECONFIG
+sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=$KUBERNETES_VERSION
+minikube update-context --profile=minikube
+sudo chown -R travis: /home/travis/.minikube/
+
+# setup samba server and deploy SMB CSI driver
+kubectl cluster-info
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n kube-system get pods -lk8s-app=kube-dns -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for kube-dns to be available";  done
+kubectl create secret generic smbcreds --from-literal username=USERNAME --from-literal password="PASSWORD"
+kubectl create -f deploy/example/smb-provisioner/smb-server.yaml
+JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n default get pods -lapp=smb-server -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for smb controller deployment to be available"; done
+bash deploy/install-driver.sh
```

Comment:
here the e2e test still installs official latest tag of driver image, I think you should use helm installation to install current driver: https://github.com/kubernetes-csi/csi-driver-smb/blob/1ca7c4879892831bcb15e8751069d31ba9678c93/Makefile#L68-L83